spec/requests/api/v1/people_controller_image_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::PeopleController do
+  describe 'GET /people/:id/image' do
+    let(:token) { create(:access_token) }
+
+    context 'when there is no image' do
+      let!(:person) { create(:profile, latest_nomis_booking_id: 123_456_789).person }
+
+      before do
+        allow(NomisClient::Image).to receive(:get).and_return(nil)
+      end
+
+      it 'returns 404' do
+        get "/api/v1/people/#{person.id}/image", params: { access_token: token.token }
+        expect(response).to have_http_status(:not_found)
+      end
+    end
+
+    context 'with an image' do
+      let!(:person) { create(:profile, latest_nomis_booking_id: 1_153_753).person }
+      let(:image_data) { File.read('spec/fixtures/Arctic_Tern.jpg') }
+
+      before do
+        allow(NomisClient::Image).to receive(:get).and_return(image_data)
+
+        get "/api/v1/people/#{person.id}/image", params: { access_token: token.token }
```

Comment:
I'd suggest to put the action into the test,
it does not matter duplication it the tests.